packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/src/models/models.ts

```diff
@@ -232,9 +231,6 @@ export enum KnownServiceApiVersions {
   V20240601 = "2024-06-01",
 }
 
-/** Alias for _EventGridCredentialUnion */
-export type _EventGridCredentialUnion = KeyCredential | TokenCredential;
```

Comment:
Including all client-level parameters generation would introduce un-necessary models and create an issue to track that: https://github.com/Azure/autorest.typescript/issues/3148.